OpalImagePicker/Source/OpalImagePickerRootViewController.swift

```diff
@@ -58,12 +58,24 @@ open class OpalImagePickerRootViewController: UIViewController {
     
     internal var fetchLimit: Int {
         get {
-            return fetchOptions.fetchLimit
+            if #available(iOS 9.0, *) {
```

Comment:
I think it would be better to make the `fetchLimit` variable only available to iOS 9 and above.